src/ddmd/dsymbolsem.d

```diff
@@ -1621,7 +1621,8 @@ extern(C++) final class Semantic3Visitor : Visitor
         if (funcdecl.parameters)
         {
             size_t nfparams = Parameter.dim(f.parameters);
-            assert(nfparams == funcdecl.parameters.dim);
+            if (funcdecl.errors)
```

Comment:
The logic is inverted here, should have a `!` prepended. But in any case, delete this line and use the one mentioned above.

Reply:
**funcdecl.errors** wasn't updated in that case. I updated the code